library/Icingadb/Setup/IcingaDbWizard.php

```diff
@@ -50,19 +50,16 @@ public function getRequirements()
     {
         $set = new RequirementSet();
 
-        $set->add(new PhpVersionRequirement([
-            'condition'     => ['>=', '7.2'],
-            'description'   => sprintf(t('Icinga DB Web requires PHP version %s.'), '7.2')
-        ]));
+        $requiredVersions = Icinga::app()->getModuleManager()->getModule('icingadb')->getRequiredLibraries();
 
         $set->add(new WebLibraryRequirement([
-            'condition'     => ['icinga-php-library', '>=', '0.9.0'],
+            'condition'     => ['icinga-php-library', $requiredVersions['icinga-php-library']],
```

Comment:
please still return the operator, but set to the empty string, to preserve compatibility with legacy Web versions